maquetado/scss/_tech.scss

```diff
@@ -43,3 +44,17 @@
     margin-bottom: 15px;
   }
 }
+
+.tech-list {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 100px;
+
+  &:not(:last-child) {
+    margin-bottom: 100px;
+  }
+
+  .tech {
```

Comment:
I recommend you avoid the nesting of classes because this has problems with the specificity. So move this property outside of here, please